### PR TITLE
latest changes according to swift 3

### PR DESCRIPTION
**Pull request by irtaneja**

### CODE REVIEW

File: Example/Tests/CoreDataStoreTests.swift [hunk 1]
```diff
@@ -346,27 +374,27 @@ class CoreDataStoreTests: XCTestCase {
...
         
         self.animalModel!.add(obj, callback: { (error, obj) -> Void in
+            print(obj)
             XCTAssert(error == nil, "Pass")
-            let num = NSNumber(double: 122233334)
-            self.animalModel?.get(id: num, params: [:], callback: { (err:NSError?, obj:AnyObject?) -> Void in
-                
+            let num = NSNumber(value: 122233334)
+            self.animalModel?.get(id: num, params: [:], callback: { (err:NSError?, obj:Any?) -> Void in
+                print(obj)
```
Comment (rtalwar26) [inline Example/Tests/CoreDataStoreTests.swift]: remove `print`

File: Example/Tests/Message.swift [hunk 2]
```diff
@@ -9,6 +9,44 @@
 import Foundation
 import CoreData
 import DataStoreKit
+fileprivate func < <T : Comparable>(lhs: T?, rhs: T?) -> Bool {
```
Comment (rtalwar26) [inline Example/Tests/Message.swift:12]: why we need `fileprivate`

File: Pod/Classes/CoreDataStack.swift [hunk 3]
```diff
@@ -9,91 +9,115 @@
...
-            ps = nil
-        } catch {
-            fatalError()
-        }
+        try _persistentStoreCoordinator!.addPersistentStore(ofType: NSSQLiteStoreType, configurationName:nil, at: url, options: options)
         
-        if (ps == nil) {
-            abort()
-        }
+        //        var ps: NSPersistentStore?
```
Comment (rtalwar26) [inline Pod/Classes/CoreDataStack.swift]: remove commented code

File: Pod/Classes/CoreDataStore.swift [hunk 4]
```diff
@@ -204,7 +205,7 @@ public class CoreDataStore<T where T:ObjectCoder>:ModelProtocol{
            
         let managedObject = results![0]
```
Comment (rtalwar26) [inline Pod/Classes/CoreDataStore.swift:205]: check `results.count` like we are doing in `put`

File: Example/Tests/Message.swift (hunk 2 above)
Comment (irtaneja) [reply Example/Tests/Message.swift:12]: this was automatically added when the code was converted to swift 3 using xcode converter

File: Example/Tests/CoreDataStoreTests.swift (hunk 1 above)
Comment (irtaneja) [reply Example/Tests/CoreDataStoreTests.swift]: removed

File: Pod/Classes/CoreDataStack.swift (hunk 3 above)
Comment (irtaneja) [reply Pod/Classes/CoreDataStack.swift]: removed

File: Pod/Classes/CoreDataStore.swift (hunk 4 above)
Comment (irtaneja) [reply Pod/Classes/CoreDataStore.swift:205]: done

**Pull request merged**